save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/demo/NewDemoToolRequest.kt

```diff
@@ -0,0 +1,12 @@
+package com.saveourtool.save.demo
+
+/**
+ * @property organizationName name of an organization on GitHub that maintains the tool
+ * @property projectName name of repo on GitHub
+ * @property vcsTagName release tag that defines the version to be fetched
+ */
+data class NewDemoToolRequest(
```

Comment:
I guess it should be marked as `@Serializable`

Reply:
`NewDemoToolRequest` is passed from backend to demo service, so it seems only Jackson serializes it. Do we need `@Serializable` for now?

Reply:
Will add it later if it is needed

Reply:
yeap, forgot about Jackson in backend